lib/rust/parser/generate-java/java/org/enso/syntax2/UnsupportedSyntaxException.java

```diff
@@ -0,0 +1,14 @@
+package org.enso.syntax2;
+
+public final class UnsupportedSyntaxException extends Exception {
+    protected final Tree tree;
```

Comment:
Ups, not `protected`. What shall `protected` mean in `final` class anyway? One can make it `private` now, when there is the `getTree` getter.